com.sudovanilla.penpot-desktop.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: penpot-desktop.desktop
+rename-icon: penpot-desktop.png
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/latest/Penpot-Desktop.AppImage
```

Comment:
you should NOT download "latest"

This is likely the continuous build and the checksum will change each time

You should download an actual release.

This also applies for the aarch64 binary.

Reply:
Now I think of it, that was a bad idea from the start.
This will be switched to a fixed version, instead of latest. 
And you are right, this would be a continuous build, resulting to a different checksum.